From fbbdfba3d22db381ea22a915b311af823924eeb9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Giedr=C4=97=20=C5=A0imk=C5=ABnait=C4=97?=
 <giedre.s@8devices.com>
Date: Fri, 14 Nov 2025 06:26:54 +0000
Subject: [PATCH] ath10k: add 5885 MHz 5G channel
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change made based on ath11k driver mac.c ath11k_5ghz_channels[] 5G
channels list.

drivers/net/wireless/ath/ath11k/mac.c
Copyright (c) 2018-2019 The Linux Foundation. All rights reserved.
Copyright (c) 2021-2023 Qualcomm Innovation Center, Inc. All rights reserved.

Upstream-Status: Pending
Signed-off-by: Giedrė Šimkūnaitė <giedre.s@8devices.com>
---
 drivers/net/wireless/ath/ath10k/core.h | 4 ++--
 drivers/net/wireless/ath/ath10k/mac.c  | 1 +
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/ath/ath10k/core.h b/drivers/net/wireless/ath/ath10k/core.h
index cb2359d2ee0b..4b11a88fbd36 100644
--- a/drivers/net/wireless/ath/ath10k/core.h
+++ b/drivers/net/wireless/ath/ath10k/core.h
@@ -38,8 +38,8 @@
 #define WMI_READY_TIMEOUT (5 * HZ)
 #define ATH10K_FLUSH_TIMEOUT_HZ (5 * HZ)
 #define ATH10K_CONNECTION_LOSS_HZ (3 * HZ)
-#define ATH10K_NUM_CHANS 41
-#define ATH10K_MAX_5G_CHAN 173
+#define ATH10K_NUM_CHANS 42
+#define ATH10K_MAX_5G_CHAN 177
 
 /* Antenna noise floor */
 #define ATH10K_DEFAULT_NOISE_FLOOR -95
diff --git a/drivers/net/wireless/ath/ath10k/mac.c b/drivers/net/wireless/ath/ath10k/mac.c
index 655fb5cdf01f..ee67e5a5562b 100644
--- a/drivers/net/wireless/ath/ath10k/mac.c
+++ b/drivers/net/wireless/ath/ath10k/mac.c
@@ -9542,6 +9542,7 @@ static const struct ieee80211_channel ath10k_5ghz_channels[] = {
 	CHAN5G(165, 5825, 0),
 	CHAN5G(169, 5845, 0),
 	CHAN5G(173, 5865, 0),
+	CHAN5G(177, 5885, 0),
 	/* If you add more, you may need to change ATH10K_MAX_5G_CHAN */
 	/* And you will definitely need to change ATH10K_NUM_CHANS in core.h */
 };
